p2panda-sync/src/log_sync.rs

```diff
@@ -319,25 +347,24 @@ mod tests {
     }
 
     #[derive(Clone, Debug)]
-    struct LogIdTopicMap(HashMap<TopicId, Vec<String>>);
+    struct LogHeightTopicMap(HashMap<String, Logs<u64>>);
```

Comment:
Very cool, that concept is so nice now and that simple mapper comes in handy for many use cases already ✨